Automattic/jetpack
# Newsletter: Add byline toggles

This PR adds toggle for different byline options that we could send in the newsletter.
See for more context. p1HpG7-t6V-p2 

## Proposed changes:
Add toggles for gravatar, author email, post date to be added for the email byline.

See:

<img width="1104" alt="Screenshot 2024-06-19 at 10 39 53 PM" src="https://github.com/Automattic/jetpack/assets/115071/b5478b1a-29b9-4e0f-a71f-7611b613929a">

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:
Visit /wp-admin/admin.php?page=jetpack&enable-byline=true#/newsletter 

To see the new toggles. 
The .com side of things is not in place just yet. So the emails will not go out with this new format. 
Interact with them and notice that they work as expected. The data is should be saved and be shown a page refresh. 
By default you should see all the toggles on.
